index.html

```diff
@@ -83,13 +83,36 @@ <h3>Сувениры</h3>
                 </ul>
             </section>
             
-            <section class="search">
-                <h2>Поиск гостиницы в Седоне</h2>
+            <section class="search-title">
+                <h2 class="visually-hidden">Поиск гостиницы в Седоне</h2>
                 <b>Заинтересовались?</b>
                 <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
                 <a class="button" href="catalog.html">Поиск гостиницы в Седоне</a>
                 <img src="img/map.jpg" width="1200" height="594" alt="Где находится Седона">
-            </section>       
+            </section>  
+            
+            <section class="modal search-modal">
+                <h2 class="visually-hidden">Форма поиска гостиниц в Седоне</h2>
+                <form class="search-form" action="https://echo.htmlacademy.ru" method="get">
+                    <p class="search-criteria">
+                        <label for="arrival-date">Дата заезда:</label>
+                        <input class="visually-hidden criteria-input" type="text" name="arrival-date" id="arrival-date" value="" placeholder="24 апреля 2017">
```

Comment:
Если у поля нет значения по-умолчанию, прописывать атрибут value не нужно.

Reply:
В ТЗ сказано, что календарик кликабельный - на него будет вешаться событие открытия календаря под полем. Соответственно это кнопка.
```suggestion
                        <input class="visually-hidden criteria-input" type="text" name="arrival-date" id="arrival-date" value="" placeholder="24 апреля 2017"><button  type="button">Выбрать дату</button>
```